woodwork/accessor_utils.py

```diff
@@ -209,7 +209,7 @@ def wrapper(self, *args, **kwargs):
             )
             raise WoodworkNotInitError(msg)
         diff_cols = set(self._dataframe.columns).difference(
-            set(self._schema.columns.keys())
+            set(self._schema.columns.keys()),
```

Comment:
Is this comma notation part of py3.10?

Reply:
Nope but I added this for this reason:
https://github.com/asottile/add-trailing-comma#multi-line-method-invocation-style----why